pyerrors/correlators.py

```diff
@@ -268,8 +268,8 @@ def GEVP(self, t0, ts=None, state=0, sorted_list=None):
             G0, Gt = np.empty([self.N, self.N], dtype="double"), np.empty([self.N, self.N], dtype="double")
             for i in range(self.N):
                 for j in range(self.N):
-                    G0[i, j] = self.content[t0][i, j].value
-                    Gt[i, j] = self.content[ts][i, j].value
+                    G0[i, j] = self.matrix_symmetric().content[t0][i, j].value
+                    Gt[i, j] = self.matrix_symmetric().content[ts][i, j].value
```

Comment:
It would be more efficient to call `matrix_symmetric` only once before the loops instead of symmetrizing every single time.